lib/vet360/models/address.rb

```diff
@@ -113,6 +113,12 @@ class Address < Base
         address.validates :province, absence: true
       end
 
+      def zip_plus_four
+        return if zip_code.blank?
+
+        [zip_code, zip_code_suffix].compact.join('-')
```

Comment:
Can `zip_code_suffix` be an empty string (i.e. '')?  If so, should probably add a conditional here.

Reply:
`zip_code_suffix` can be `nil`, but shouldn't show up as an empty string. The compact should catch the `nil`.